exercises/protein-translation/protein_translation_test.py

```diff
@@ -35,25 +37,34 @@ def test_identifies_stop_codons(self):
         for codon in ['UAA', 'UAG', 'UGA']:
             self.assertEqual('STOP', of_codon(codon))
 
-    def test_translates_rna_strand_into_correct_protein(self):
+    def test_translates_rna_strand_into_correct_protein_list(self):
         strand = 'AUGUUUUGG'
         expected = ['Methionine', 'Phenylalanine', 'Tryptophan']
         self.assertEqual(expected, of_rna(strand))
```

Comment:
In an effort to maintain consistency with the spec, `of_rna()` should probably be renamed to `proteins()`.